ui/analyse/src/study/playerBars.ts

```diff
@@ -13,7 +13,7 @@ export default function (ctrl: AnalyseCtrl): VNode[] | undefined {
   if (!study) return;
   const relayPlayers = study.relay?.players;
 
-  const players = study.currentChapter().players,
```

Comment:
I disagree with this fix. The type of `currentChapter()` is `ChapterPreview` so we should not have to check it against `null` or `undefined`.

Either the type of that function or its implementation should instead be fixed.